AsyncDisplayKit/ASControlNode.mm

```diff
@@ -492,7 +493,60 @@ - (void)layout
       intersectSuperlayer = intersectLayer.superlayer;
     }
     
-    _debugHighlightOverlay.frame = [intersectLayer convertRect:intersectRect toLayer:layer];
+    CGRect finalRect = [intersectLayer convertRect:intersectRect toLayer:layer];
+    UIColor *fillColor = [[UIColor greenColor] colorWithAlphaComponent:0.4];
+  
+    // determine which edges were clipped
```

Comment:
This is cool, but the code is pretty generically applicable and probably should not live in control mode in the long term.

Could you think about a way to generalize it and look around for a file that might be appropriate to contain this kind of functionality, or perhaps we can create an image utility file where we could also implement a simple rounded rectangle stretchable image  generator?